node/src/accountant/mod.rs

```diff
@@ -2121,15 +1584,16 @@ mod tests {
             gwei_amount: 10000,
         };
         let more_money_received_params_arc = Arc::new(Mutex::new(vec![]));
+        let payable_dao = PayableDaoMock::new().non_pending_payables_result(vec![]);
         let receivable_dao = ReceivableDaoMock::new()
             .more_money_received_parameters(&more_money_received_params_arc)
             .more_money_received_result(Ok(()));
         let accountant = AccountantBuilder::default()
-            .bootstrapper_config(bc_from_ac_plus_earning_wallet(
-                make_populated_accountant_config_with_defaults(),
-                earning_wallet.clone(),
-            ))
-            .payable_dao(PayableDaoMock::new().non_pending_payables_result(vec![]))
+            .bootstrapper_config(bc_from_earning_wallet(earning_wallet.clone()))
+            .payable_dao(payable_dao)
```

Comment:
It used to be true that if you didn't have to supply any specific DAO, the Accountant builder built it while using the simplest, and so empty, mocks. If the comment one step above is right and if the Accountant still functioned in a similar way it would be nice to just omit these three building components adding methods.    

Think of whether you could preserve such a behavior on the builder and if it were worth it.